src/main/java/pl/hk/zadanie_22_1/Sender.java

```diff
@@ -0,0 +1,39 @@
+package pl.hk.zadanie_22_1;
+
+public class Sender {
```

Comment:
Nazwa `Sender` wydaje mi się nieodpowniednia ze względu na to, że to ne jest tlyko informacja o nadawcy, ale również o treści samej wiadoości